examples/sri-1/components/workgroup/invitation/invitation.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Agreement } from "../storage/agreement";
+import { BpiSubject } from "../identity/bpiSubject";
+export interface IInvitation {
```

Comment:
Please provide reasoning.

Reply:
this is because of the default variable merging that TS compiler does. In the beginning it might not seem to be a big problem, but it can lead into issues as the project grows.